doc/polyglossia.tex

```diff
@@ -302,7 +302,7 @@ \subsection{Global options}
 \begin{itemize}
 	\item \TB{babelshorthands}\new{v1.1.1} globally activates \pkg{babel}
           shorthands whenever available. Currently shorthands are implemented for
-          Catalan, Czech, Dutch, Finnish, German, Italian, Mongolian, and Russian.
+          Catalan, Czech, Dutch, Finnish, German, Italian, Latin, Mongolian, and Russian.
```

Comment:
Could you please add Slovak here? I forgot that in my change, and I don't want to interfere with yours.

Reply:
Done.